scripts/scr_weapon/scr_weapon.gml

```diff
@@ -883,7 +883,7 @@ global.weapons = {
 		"range": 3.1,
 		"spli": 1,
 		"arp": 1,
-		"tags": ["plasma", "pistol"]
+		"tags": ["plasma","energy", "pistol"]
```

Comment:
Is it necessary to have plasma AND energy tags?
We have plasma and las, isn't it possible to just check for them for the advantage?

Reply:
I would've gotten rid of Plasma as well but its already tied to an STC bonus